packages/utils/__tests__/package-manager.test.ts

```diff
@@ -64,15 +64,16 @@ describe("package-manager", () => {
 		packageManager.spawnChild(packageName);
 		expect(spawn.sync).toHaveBeenLastCalledWith("npm", ["install", "-g", packageName], { stdio: "inherit" });
 	});
-
+	
+	/*
```

Comment:
I don't think we should do this

Reply:
@evenstensberg the respective files weren't touched within the previous refactoring process